### ISSUE TITLE
camxproj.txt

### ISSUE THREAD

**Issue by lizadams** [CEMPD/VERDI#28]
VERDI_1.5.0 doesn't seem to be reading the camxproj.txt file once a user edits it. The location of the camxproj.txt is the same directory as where a UAM binary file is located.
There is a test UAM file under the following directory VERDI_1.5.0/data/CAMx
The previous version of VERDI 1.4.1 does correctly read this projection information from the camxproj.txt and correctly draws the map if provided the 

Here an example camxproj.txt file:

# Projection parameters are based on IOAPI.  For details, see www.baronsams.com/products/ioapi
GDTYP=2
P_ALP=33.0
P_BET=45.0
P_GAM=-97.0
XCENT=-97.0
YCENT=40.0

**Comment by lizadams** [comment]
This code was submitted to netcdf and was included in earlier versions according to this CHANGES.txt file netcdf/ncSrc-4.2/cdm/CHANGES.txt:  UAMIVServiceProvider looks for camxproj.txt for config info

**Comment by lizadams** [comment]
on killdevil this file is located here: 
/nas01/depts/ie/cempd/apps/VERDI/netcdf/netcdf/ncSrc-4.2/cdm/src/main/java/ucar/nc2/iosp/uamiv/UAMIVServiceProvider.java

**Comment by lizadams** [comment]
It is also under the new version of thredds - which we are using in verdi.
/nas01/depts/ie/cempd/apps/VERDI/thredds/cdm/src/main/java/ucar/nc2/iosp/uamiv/UAMIVServiceProvider.java
It looks like this needs the **projpath** to be set correctly - perhaps this is the problem.
   if (lastIndex <= 0)
      lastIndex = projpath.lastIndexOf('/');
    if (lastIndex > 0)
      projpath = projpath.substring(0, lastIndex);
    projpath = projpath + File.separator + "camxproj.txt";
    File paramFile = new File(projpath);
    if (paramFile.exists()) {
      BufferedReader br = new BufferedReader(new FileReader(paramFile));
      while ((thisLine = br.readLine()) != null) {
        if (thisLine.substring(0, 1) != "#") {
          key_value = thisLine.split("=");
          if (key_value[0].equals("GDTYP")) {
            gdtyp = Integer.parseInt(key_value[1]);
            lgdtyp = true;

**Comment by lizadams** [comment]
I noticed this bug when doing the SMOKE training and trying to visualize the CAMx SMOKE output.  
It may have been a permissions problem??  VERDI will create a camxproj.txt file in the same directory as the uam output file, but if the person doesn't have permission to create this file then it won't.  I just tried to reproduce this bug on my mac running VERDI 1.5.0 and I was able to edit the camxproj.txt file and have VERDI create a fast tile plot with the correct underlying map.  Going to close this issue.
